packages/odyssey-storybook/.storybook/components/MuiThemeDecorator.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { ThemeProvider as MuiThemeProvider } from "@mui/material/styles";
+import { ThemeProvider } from "@storybook/theming";
+import { theme } from "@okta/odyssey-react-mui";
+import type { DecoratorFn } from "@storybook/react";
+
+export const MuiThemeDecorator: DecoratorFn = (Story) => (
+  <MuiThemeProvider theme={theme}>
+    <ThemeProvider theme={theme}>
+      <Story />
+    </ThemeProvider>
+  </MuiThemeProvider>
```

Comment:
Are we needing to use both `MuiThemeProvider` in consuming apps? Why isn't it integrated into `ThemeProvider`? As a consumer of Odyssey, I'm not going to know which components are and are not Material-UI because I'm not looking at their source code.

Reply:
This is only necessary for Storybook. Additionally, note that this work only affects the unreleased Odyssey MUI components.

Reply:
mui is broken for custom themes: https://github.com/mui/material-ui/issues/24282#issuecomment-952211989